content/r-package-dev/Documentation.Rmd

```diff
@@ -262,11 +304,17 @@ etc ...
 
 ```
 
-Save your new R Markdown file. The next time you build your package, your vignette should now be available!
+Save your new R Markdown file. To get the vignette to show up in your R package, you then need to:
+
+```{r eval=FALSE}
+devtools::build_vignettes()
+```
+
+This will create the html output, an R script that is just the executable R code in the vignette, and a copy of the markdown file, moving all 3 of those files into the inst/doc folder.
```

Comment:
So here's the thing. If you have a .Rmd file in the vignette's folder, and you only count on roxygen...it will create the .R and .html files, but you still have to then manually move them from the vignette folder to the inst/doc folder. If you just use `devtools::build_vignette()`, it does both of those steps. I guess there's also a way to use a `make` file in your vignette folder, but I'd still vote the devtools function is the easiest.